src/NuGet.Services.Metadata.Catalog.Monitoring/Model/PackageRegistrationLeafMetadata.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Newtonsoft.Json;
+using NuGet.Protocol;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    /// <summary>
+    /// The metadata for a particular package in its registration leaf.
+    /// </summary>
+    public class PackageRegistrationLeafMetadata
+    {
+        public PackageRegistrationLeafMetadata()
+        {
+        }
+
+        public PackageRegistrationLeafMetadata(V2FeedPackageInfo package)
+        {
+            Listed = package.IsListed;
+            PackageContent = package.DownloadUrl;
+            Published = package.Published;
+        }
```

Comment:
`PackageContent` is the name of the field in the registration blobs. This constructor converts a `V2FeedPackageInfo` into a form that can be directly compared to a `PackageRegistrationLeafMetadata`.

Perhaps I should
a) add a comment here
b) remove this and compare the values directly (given that I'm already planning to remove `ComparisonUtils` which was the only reason a single class was required)